src/util/Injected.js

```diff
@@ -40,6 +40,7 @@ exports.ExposeStore = (moduleRaidStr) => {
     window.Store.Validators = window.mR.findModule('findLinks')[0];
     window.Store.VCard = window.mR.findModule('vcardFromContactModel')[0];
     window.Store.Wap = window.mR.findModule('queryLinkPreview')[0].default;
+    window.Store.queryLinkPreview = window.mR.findModule(x=>x.default && x.default.cache && x.default.name === 'n')[0].default;
```

Comment:
I think this is an unreliable way to find this module, can we do it in another way?

Reply:
I tend to agree, usually we find modules through common method names within the same module, so for example I used deprecatedSendIq to use deprecatedCastStanza. @shirser121 please figure this one out